Source/WTF/wtf/text/cf/StringImplCF.cpp

```diff
@@ -105,7 +105,9 @@ namespace StringWrapperCFAllocator {
 
     static CFAllocatorRef allocator()
     {
+        ASSERT(isMainThread());
         static NeverDestroyed allocator = [] {
+            initializeMainThread();
             CFAllocatorContext context = { 0, nullptr, retain, release, copyDescription, allocate, reallocate, deallocate, preferredSize };
             return adoptCF(CFAllocatorCreate(nullptr, &context));
         }();
```

Comment:
Maybe we should avoid race condition here? Like,
```
static std::once_flag onceKey;
LazyNeverDestroyed<...> allocator;
std::call_once(onceKey, [&] {
    allocator.construct(...);
});
return allocator.get();
```

Reply:
This is guarded by a `isMainThread()` check so it should only be called from the main thread.

Reply:
OK